sdk/analytics/src/main/java/com/klaviyo/analytics/Klaviyo.kt

```diff
@@ -54,6 +62,7 @@ object Klaviyo {
 
         Registry.register<State>(KlaviyoState())
         Registry.register<StateSideEffects>(StateSideEffects())
+        Registry.get<State>().apiKey = apiKey
```

Comment:
trigger an apiKey `braodcastChange` if necessary, results in registering a push token